### ISSUE TITLE
Notifications (for stdout results) fold, so I can't see the result.

### ISSUE THREAD

**Issue by Liverm0r** [liquidz/vim-iced#393]
I gues I have to put an `autocmd` for `set nofoldenable`, but can't figure out what event for autocmd should be. Or may be there is another solution for this.

![image](https://user-images.githubusercontent.com/14236531/161032845-fca4f820-6b12-493c-8b52-74d92b13f14e.png)

**Comment by Liverm0r** [comment]
Or may be the problem is in that the notification reasult is shown in one line...

**Comment by liquidz** [comment]
@Liverm0r Thanks for your reporting!

Hmm, I've never seen such a behavior.

Could you tell me a minimal configuration to reproduce your problem?
cf. https://liquidz.github.io/vim-iced/#minimal_configuration

**Comment by Liverm0r** [comment]
The only line I've found on my vimrc that matters (withtout it works fine) is this one
```vim
set fdm=syntax
```

Interesting part. I can reproduce the problem on my linux manjaro pc with this line and I can't reproduce it on my osx m1. Same dotfiles, same plugins.
My dotfiles ([vimrc_common](https://github.com/Liverm0r/dotfiles/blob/76278d9d85286228748ceabc3e20eaa51ef1af7b/.vimrc_common#L121)).

**Comment by Liverm0r** [comment]
![Screenshot_20220331_162733](https://user-images.githubusercontent.com/14236531/161065967-ff06a568-28a2-4cea-abec-d8a9546ebcbf.png)

```vim
set nocompatible
set encoding=utf-8
scriptencoding utf-8
filetype plugin indent on
set fdm=syntax

let mapleader=","

let g:iced_enable_default_key_mappings = v:true

set runtimepath-=$HOME/.vim
set runtimepath-=$HOME/.vim/after
set runtimepath-=$HOME/.config/nvim
set runtimepath-=$HOME/.config/nvim/after
set runtimepath-=$HOME/.local/share/nvim/site
set runtimepath-=$HOME/.local/share/nvim/site/after

set packpath=/tmp/vim-iced-test
```

**Comment by liquidz** [comment]
@Liverm0r Thanks!
I could reproduce the problem with non M1 mac and nvim (couldn't with vim interestingly)
I'll try to fix it :)

**Comment by liquidz** [comment]
@Liverm0r Just released v3.8.4 #394 
It should fix this problem ☺️

**Comment by Liverm0r** [comment]
Thank you!
